docs/references/nextjs/usage-with-older-versions.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Use Clerk with Next.js 12 and older
+description: Learn how to use Clerk with older versions of Next.js.
+---
+
+# Use Clerk with Next.js 12 and older
+
+Clerk's [prebuilt components](/docs/components/overview) which are exported from the `@clerk/nextjs` package are leveraging APIs from Next.js itself. These APIs often change between major Next.js releases. While Clerk tries to offer the highest compatibility for any supported Next.js version, for Next.js 12 and older you'll need to modify your setup.
+
+<Callout type="warning">
+  Clerk highly recommends updating your Next.js version as older versions won't receive any updates in the future. Read [their upgrade guides](https://nextjs.org/docs/pages/building-your-application/upgrading) to learn more.
+</Callout>
+
+<Steps>
+
+### Install `@clerk/nextjs` v4
+
+You need to install `^4.0.0` of `@clerk/nextjs`. Newer major versions only support Next.js 13+.
```

Comment:
Using an active voice here. Instead of "you need to", we can simply go straight to telling the user what to do with "install".

```suggestion
Install `^4.0.0` of `@clerk/nextjs`. Newer major versions of `@clerk/nextjs` only support Next.js 13+.
```